access-control-app/src/cmr/access_control/services/parameter_validation.clj

```diff
@@ -108,3 +108,10 @@
   "Throws service errors if any invalid params or values are found."
   [params]
   (validate-params params :user-token))
+
+(defn validate-s3-buckets-params
+  "Throws service errors if any invalid params or values are found."
+  [params]
+  (validate-params params :user_id :provider)
+  (when-not (contains? params :user_id)
```

Comment:
change this to something like `(when (clojure.string/blank? (:user_id params))` to be more robust.